File: python/_8.py
def read_file(path: str) -> str | None:
    try:
        with open(path) as file:
            out_put = file.read()
            return out_put
    except:
        return None



result = read_file("/mnt/d/Project/devfest/python/_1.py")

if result:
    print(f"{result}")
else:
    print("nothing")

File: rust/_8.rs
use std::fs::File;
use std::io::Read;

fn read_file(path: &str) -> Option<String> {
    match File::open(path) {
        Ok(mut file) => {
            let mut out_put = String::new();
            file.read_to_string(&mut out_put);
            Some(out_put)
        }
        Err(_) => None,
    }
}

fn main() {
    if let Some(result) = read_file("/mnt/d/Project/devfest/rust/_1.rs") {
        println!("{result}");
    } else {
        println!("nothing");
    }
}
